artemis-commons/src/main/java/org/apache/activemq/artemis/utils/bean/MetaBean.java

```diff
@@ -123,10 +123,9 @@ public void parseToJSON(T object, JsonObjectBuilder builder, boolean ignoreNullA
                }
             } else if (type == Boolean.class) {
                builder.add(name, (Boolean) value);
+            /* This is the same as the default, but keeping this in case we have to deal with Enums in the future
             } else if (Enum.class.isAssignableFrom(type)) {
-               // I know this is the same as the default else clause further down
-               // but i wanted to have a separate branch in case we have to deal with it later
-               builder.add(name, String.valueOf(value));
+               builder.add(name, String.valueOf(value));*/
```

Comment:
Ick. Maybe better just deleted at this point?

Reply:
I'm fine with deleting this, but I left it in to be conservative.

Reply:
@clebertsuconic, are you OK with deleting this?